src/Apps/Search/Components/SearchResultsSkeleton/FilterSidebar.tsx

```diff
@@ -1,34 +1,30 @@
-import { Box, color, Separator } from "@artsy/palette"
+import { Box, color, Flex, Separator } from "@artsy/palette"
 import React from "react"
 
 const FilterSidebarSection: React.SFC<any> = props => {
   return (
     <>
       <Box
-        width={100}
-        height={8}
-        style={{ marginTop: "20px", backgroundColor: color("black10") }}
-      />
-      <Box
-        width={120}
-        height={8}
-        style={{ marginTop: "20px", backgroundColor: color("black10") }}
-      />
-      <Box
-        width={120}
-        height={8}
-        style={{ marginTop: "20px", backgroundColor: color("black10") }}
-      />
-      <Box
-        width={120}
-        height={8}
-        style={{ marginTop: "20px", backgroundColor: color("black10") }}
-      />
-      <Box
-        width={120}
-        height={8}
-        style={{ marginTop: "20px", backgroundColor: color("black10") }}
+        width={80}
+        height={14}
+        style={{ marginBottom: "15px", backgroundColor: color("black10") }}
```

Comment:
@kierangillen - `marginBottom` goes outside of our design system. All values need to be constrained within palette's [theme](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx) file so that we don't sucumb to design drift. That means often rounding up or rounding down from the redlines.

Reply:
Additionally, don't need to use `style` tags for props that can be pulled from `styled-system`'s API (meaning `background`): https://styled-system.com/table. In this case can use `bg`. 

If something isn't working via props, it's important to look directly at the source of truth -- [`Box`](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/Box/Box.tsx#L60-L74) from within Palette.

Reply:
That makes sense – how can we tell whether a component _should_ use `marginBottom` or otherwise step outside our design system? Is there ever a legitimate case for this? The search results skeleton here is pretty unusual, for example, and I can see it needing more than our Palette theme can offer.

Reply:
There's typically very few cases where we should step out of the design system, as it was built entirely on constraints. But I do see what you mean re the skeleton since that more graphical and overlays what will eventually be actual UI 👍

Reply:
@damassi I would argue that this skeleton may fall in an area outside our normal design system, as I'm trying to achieve pixel perfect mirroring of the actual page. For this reason font sizes and margins may visually be different given the the line height of the fonts on the actual rendered page vs where they fall visually.

Reply:
I can update the rest of the code to not use the `styled` attribute, that was code that I opted to not update where I wasn't touching or making changes. But best to take the time to update!

Reply:
Side-note: with `marginBottom` we should still use `mb` as we do elsewhere, as a top-level prop; styled-system's lookup system will first check  the theme, and if it can't find the lookup key there it will convert the value to pixels, so all values are basically permitted.

Reply:
> I would argue that this skeleton may fall in an area outside our normal design system

Yup agree.